portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -17,10 +22,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String email = request.getParameter("email");
     String message = request.getParameter("message");
 
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    KeyFactory keyFactory = datastore.newKeyFactory().setKind("Contact");
+    FullEntity contactEntity =
+        Entity.newBuilder(keyFactory.newKey())
+            .set("name", name)
+            .set("email", email)
+            .set("message", message)
+            .build();
+    datastore.put(contactEntity);
+    
     // Print the value so you can see it in the server logs.
     System.out.println(name + " submitted \"" + message + "\" from " + email);
 
     // Write the value to the response so the user can see it.
     response.getWriter().println("You submitted: \"" + message + "\" to me :)" );
```

Comment:
Stretch opportunity: also return a button or link that people can click on to get back to the /contact.html page.  Currently I need to hit the back button to do so